furniture_store/data/pos_category.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="pos_category_1" model="pos.category">
```

Comment:
You need to add products in the relevant pos categories, otherwise it won't be useful

Reply:
okay